app/views/follow_requests/show.html.erb

```diff
@@ -1,5 +1,3 @@
-<p style="color: green"><%= notice %></p>
```

Comment:
Nice job removing all alerts/notices from other views since you included them in the flash partial!